action/sealedenvelope.go

```diff
@@ -161,3 +161,56 @@ func (sealed *SealedEnvelope) VerifySignature() error {
 	}
 	return nil
 }
+
+// LoadProtoWithChainID use chainID while loading protobuf
+func (sealed *SealedEnvelope) LoadProtoWithChainID(pbAct *iotextypes.Action) error {
+	if pbAct == nil {
```

Comment:
same here

Reply:
same here, this code will be reverted after 180 HF, so would like to keep it simple

Reply:
as well as this function